spread/lxd.go

```diff
@@ -589,6 +600,7 @@ func sshReloadCommand(distro Distro) []string {
 func (p *lxdProvider) tuneSSH(name string, distro Distro) error {
 	cmds := [][]string{
 		{"sed", "-i", `s/^\s*#\?\s*\(PermitRootLogin\|PasswordAuthentication\)\>.*/\1 yes/`, "/etc/ssh/sshd_config"},
+		{"/bin/sh", "-c", "[ -d /etc/ssh/sshd_config.d ] && echo PermitRootLogin yes\\\\nPasswordAuthentication yes > /etc/ssh/sshd_config.d/10-spread.conf"},
```

Comment:
This doesn't get overridden by the later `60-cloudinit-thingy.conf`?

Reply:
I guess it must not, because WLCS CI is happy.

Reply:
Yeah, it's surprising, but it's the first one that's respected.